metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorDatabaseService.java

```diff
@@ -121,26 +121,32 @@ public void update(@Nonnull @NonNull final ConnectorContext context,
         final QualifiedName databaseName = databaseInfo.getName();
         try {
             this.metacatHiveClient.alterDatabase(databaseName.getDatabaseName(),
-                    hiveMetacatConverters.fromDatabaseInfo(databaseInfo));
+                hiveMetacatConverters.fromDatabaseInfo(databaseInfo));
         } catch (NoSuchObjectException exception) {
             throw new DatabaseNotFoundException(databaseName, exception);
         } catch (MetaException | InvalidObjectException exception) {
             throw new InvalidMetaException(databaseName, exception);
         } catch (TException exception) {
             throw new ConnectorException(
-                    String.format("Failed updating hive database %s", databaseName), exception);
+                String.format("Failed updating hive database %s", databaseName), exception);
         }
     }
 
     /**
      * {@inheritDoc}.
      */
     @Override
-    public DatabaseInfo get(@Nonnull @NonNull final ConnectorContext requestContext,
-                            @Nonnull @NonNull final QualifiedName name) {
+    public DatabaseInfo get(
+        @Nonnull @NonNull final ConnectorContext requestContext,
+        @Nonnull @NonNull final QualifiedName name
+    ) {
         try {
             final Database database = metacatHiveClient.getDatabase(name.getDatabaseName());
-            return hiveMetacatConverters.toDatabaseInfo(name, database);
+            if (database != null) {
```

Comment:
@ajoymajumdar @zhljen This is the associated null check and exception throw change from my other comment for the test behavior change.